### PR TITLE
SmallRye Fault Tolerance 4.3.1

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

File: _posts/2020-09-29-fault-tolerance-4-3-1.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+:page-layout: post
+:page-title: Fault Tolerance 4.3.1
+:page-synopsis: SmallRye Fault Tolerance 4.3.1 released!
+:page-tags: [announcement, microprofile]
+:page-date: 2020-09-29 14:00:00.000 +0100
+:page-author: lthon
+
+= Fault Tolerance 4.3.1
+
+Today, we announce a release of SmallRye Fault Tolerance 4.3.1.
```
Comment (radcortez) [inline _posts/2020-09-29-fault-tolerance-4-3-1.adoc:10]:
```suggestion
Today, we announce the release of SmallRye Fault Tolerance 4.3.1.
```

File: _posts/2020-09-29-fault-tolerance-4-3-1.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This is a small release with a couple of bug fixes and one API deprecation.
+It is a safe upgrade for everyone using SmallRye Fault Tolerance 4.3.0.
+
+In the previous release, we introduced an experimental API for observing circuit breaker state changes.
+The `CircuitBreakerStateChanged` event type is now deprecated and will be removed in a subsequent release.
+At the same time, we will introduce a new API for accessing circuit breaker maintenance features: observing its current state, and resetting to the initial state.
```
Comment (radcortez) [inline _posts/2020-09-29-fault-tolerance-4-3-1.adoc:16]:
```suggestion
At the same time, we will introduce a new API for accessing circuit breaker maintenance features: observing its current state and resetting to the initial state.
```

**Review by radcortez — APPROVED** [review]

**Comment by radcortez** [comment]
Just a few minor suggestions if you want to take them :)

Maybe include a link to the release notes as well?

**Comment by Ladicek** [comment]
Good point with the link, I'll do that.

**Comment by Ladicek** [comment]
Done.

**Review by radcortez — APPROVED** [review]

**Pull request merged**